src/main/java/uk/gov/hmcts/reform/managecase/client/datastore/DataStoreApiClient.java

```diff
@@ -33,4 +35,7 @@ CaseUserRoleResource getCaseAssignments(@RequestParam("case_ids") List<String> c
     @DeleteMapping(value = CASE_USERS, consumes = APPLICATION_JSON_VALUE)
     void removeCaseUserRoles(@RequestBody CaseUserRolesRequest userRolesRequest);
 
+    @GetMapping(CASES_WITH_ID)
```

Comment:
What is stopping us to remove ES api completely "searchCases" ? 
(also from wiremock)

Reply:
Resolved in https://github.com/hmcts/aac-manage-case-assignment/pull/173/commits/2fd9ff1c23b960edebd4606cfb08c91cccb83ce4